### ISSUE TITLE
Replicate results

### ISSUE THREAD

**Issue by alimontarey** [weepiess/StyleFlow-Content-Fixed-I2I#12]
Hi, first thanks for your nice work!
I saw that you applied your method to image dehazing and it outputs nice results. The dataset that you use has ground truth clean images.

 I wonder if you used only half of the hazy images as the corrupted set and only half of the ground truth as the clean set so the setting is really unpaired or use all of the images in both sets? 

Also, can you please share the config file for the dehazing experiments? or at least the most important hyperparameters like the content_weight, style_weight, smoothness loss used, keep_ratio..?

Thanks.

**Comment by WeichenFan** [comment]
Hi, thank you for your interest. The results shown in the picture are from the model trained with paired settings. However, we find that our model does support unpaired settings. For dehazing experiments, we suggest using k=1.0.
